src/components/Book.js

```diff
@@ -0,0 +1,28 @@
+import React from 'react';
+import { PropTypes } from 'prop-types';
+
+const Book = ({ book }) => (
+  <div className="book">
+    <h3>
+      BOOK NAME :
+      {book.title}
+    </h3>
+    <p>
+      ISBN ID :
+      {book.id}
```

Comment:
### [OPTIONAL]
- It's logical for the ID to come before the title. Making this change will improve the UI/UX experience of the app.